setup.py

```diff
@@ -67,7 +67,7 @@
         "pytest-mock",
     ]
     doc_requirements = [
-        "sphinx",
+        "sphinx<8.2.0",
```

Comment:
(temporarily) required b/c 8.2.0 was released 1 hour ago and [broke things](https://github.com/sphinx-doc/sphinx/issues/13362).

Reply:
It did - I remove the pin in the next pr